### ISSUE TITLE
suggestion limited garnade buy per round!

### ISSUE THREAD

**Issue by Escap3d** [s1lentq/ReGameDLL_CS#1018]
Can you add an option for limited grenade purchases, such as:

Flashbang limited to 2 per round
HE grenade limited to 1 per round
Smoke grenade limited to 1 per round

Often during matches, players buy too many grenades and spam them. It would be more advanced if there was a limit on purchases, so players can't buy too many smoke grenades either.

**Comment by Vaqtincha** [comment]
Set mp_buytime 0.25

**Comment by Escap3d** [comment]
Is there any solution where instead of reducing the buy time, I want to limit grenade purchases after each spawn?"


> Set mp_buytime 0.25

**Comment by StevenKal** [comment]
Use a plugin, this already exists.

**Comment by Escap3d** [comment]
> Use a plugin, this already exists.

Give me if u have plugin? And link

**Comment by LLGunknown** [comment]
> Use a plugin, this already exists.


Yes, you can use a plugin, but if you use grenade drop cvars (e.g. dead players drop all grenades), living players cannot use those grenades, because the plugin limits them.
